07-statistical-testing.Rmd

```diff
@@ -1 +1,570 @@
+
+Additionally, as with the t-test function, both `svygofchisq()` and `svychisq()` have the `na.rm` argument.  This argument defaults to `FALSE`, however, unlike the t-test function if any data is missing the $\chi^2$ tests will assume that `NA` is a category and will include it in the calculation. Throughout this chapter we will always set `na.rm=TRUE`, but before analyzing the survey data, make sure to review the notes provided in Chapter \@ref(c03) to better understand how to handle missing data.
+
+
+### Examples {#stattest-chi-examples}
+
+Let's walk through a few examples using the ANES data.  See Section \@ref(stattest-setup) above to set up the design object.
+
+#### Example 1: Goodness of Fit Test {.unnumbered #stattest-chi-ex1}
+
+ANES asked respondents their highest education level.  Based on the data from the 2020 American Community Survey (ACS) 5-year estimates^[Data was pulled from data.census.gov using the S1501 Education Attainment 2020: ACS 5-Year Estimates Subject Tables], the education distribution of those 18+ in the U.S. is as follows:
+-  11% had less than High School degree
+-  27% had a High School degree
+-  29% had some college or associate's degree
+-  33% had a bachelor's degree or higher
+
+If we want to see if the weighted distribution from the ANES 2020 data matches this distribution, we could set up the hypothesis as follows:
+
+- $H_0: p_1 = 0.11, ~ p_2 = 0.27, ~ p_3 = 0.29, ~ p_4 = 0.33$
+- $H_A:$ at least one of the education levels does not match between the ANES and the ACS
+
+To conduct this in R, we first need to adjust the education variable we have on ANES (`Education`) to have the same categories as the data from the ACS.  We do this using the {forcats} package and the `fct_collapse()` function. The education data from ANES has two levels for Bachelor's Degree or Higher (Bachelor's and Graduate), so these two categories need to be collapsed into a single category to match the ACS data.  Then we will use the `svygofchisq()` function to compare the ANES data to the ACS data:
+```{r}
+#| label: stattest-chi-ex1
+anes_des_educ<-anes_des %>% 
+  mutate(Education2=
+           fct_collapse(Education,
+                        "Bachelor or Higher"=c("Bachelor's",
+                                               "Graduate")))
+
+chi_ex1<-anes_des_educ %>% 
+  svygofchisq(formula=~Education2,
+              p=c(0.11,0.27,0.29,0.33),
+              design=.,
+              na.rm=TRUE)
+
+chi_ex1
+```
+The output from the `svygofchisq()` indicates that at least one proportion from ANES does not match the ACS data ($\chi^2=$`r chi_ex1$statistic`; $p-value=$`r chi_ex1$p.value`. To get a better idea of the differences, we can use the `expected` output along with `survey_mean()` to create a comparison table:
```

Comment:
This doesn't render correctly in HTML but does ok in PDF. 

![image](https://user-images.githubusercontent.com/8659598/235373516-943a8819-12e7-4b1b-9f78-3bccd5a82833.png)

Reply:
I fixed the $ issue, but it is still displaying with scientific notation.  Thoughts on how to fix that?